lib/services/proxywatcher.go

```diff
@@ -215,7 +225,7 @@ func (p *ProxyWatcher) watch() error {
 	case <-watcher.Done():
 		return trace.ConnectionProblem(watcher.Error(), "watcher is closed")
 	case <-reloadC:
-		p.Debugf("Triggering scheduled reload.")
+		p.log.Debugf("Triggering scheduled reload.")
```

Comment:
why this change?

Reply:
Embedding `sync.RWMutex` and `logrus.Entry` trashes autocomplete for me.